packages/textmate-grammars/data/php.html.tmLanguage.json

```diff
@@ -0,0 +1,189 @@
+{
```

Comment:
It does not seem to be used, please remove if it not

Reply:
But then, when #4796 is resolved, we'll have to add it again. Are your sure we must remove it for now?

Reply:
> There should a PR against https://github.com/theia-ide/theia-php-extension which removes textmate grammars. It should be merged immediately after this PR is merged to avoid breaking clients using next.
> 
> Code looks good now in this PR.

I can merge this PR, but it looks like I'm not allowed to merge complimentary grammars removal PR https://github.com/theia-ide/theia-php-extension/pull/17 

Could you, please, merge that complimentary PR once we've merged this one?

Reply:
sure i can do, just ping me there after merging

Reply:
> But then, when #4796 is resolved, we'll have to add it again. Are your sure we must remove it for now?

you can comment on the issue that it should be added after the PR is resolved, right now i don't see a reason to commit it

Reply:
OK. Then I'm removing it from the current PR.

Reply:
I've removed packages/textmate-grammars/data/php.html.tmLanguage.json from the PR. So it looks like we only need to wait for the CQ https://dev.eclipse.org/ipzilla/show_bug.cgi?id=19509 to be resolved